z390test/src/test/groovy/org/z390/test/RunAsmDemos.groovy

```diff
@@ -0,0 +1,42 @@
+package org.z390.test
+
+import org.junit.jupiter.api.Test
+
+class RunAsmDemos extends z390Test {
+
+    var options = ['trace', 'noloadhigh',
+                   "SYSMAC(${basePath("mac")})",
+                   "SYSOBJ(${basePath("linklib")})"]
```

Comment:
Why did you add SYSOBJ? It is not in bash/runasmdemos. Adding it will clutter the linklib directory with  .OBJ files.

I agree that not putting SYSOBJ results in .OBJ files cluttering the demo directory, but better to clutter it than the linklib directory.

Reply:
It is added because TESTDC1 and TESTASC1 reference CVTTOHEX module that is assembled as part of bldlib script and the object file is part of linklib.
I can limit the scope to just the above modules as part of the link step that requires access to the object modules.